packages/@sanity/language-filter/src/SelectLanguage.tsx

```diff
@@ -94,7 +94,7 @@ const SelectLanguage = ({
   }
 
   const content = (
-    <div onKeyUp={handleKeyUp}>
+    <Box overflow="auto" sizing="border" onKeyUp={handleKeyUp}>
       <Box padding={2}>
```

Comment:
These 2 `Box` components could be merged:

```suggestion
    <Box overflow="auto" padding={2} sizing="border" onKeyUp={handleKeyUp}>
```